docusaurus/docs/reference/30-configuration/router.md

```diff
@@ -348,6 +348,7 @@ The `metrics` section controls how metrics are communicated to the connection co
 
 - `reportInterval` - (optional, 1m) the interval of time to wait between crafting a new metrics message to be sent to the controller
 - `messageQueueSize` - (optional, 10) the maximum number of buffered metrics messages allowed to be queued to send to the controller
+- `intervalAgeThreshold` - (optional, 80s) the interval at which metrics are batched for reporting
```

Comment:
is it really 80s? that's such a strange value, it feels like a typo?

Reply:
I based that on [the package default](https://github.com/openziti/metrics/blob/main/usage_registry.go#L44).